CHANGELOG.md

```diff
@@ -1,41 +1,59 @@
 # App Center SDK for .NET Change Log
 
-## Version 2.1.0 (Under development)
+## Version 2.1.0
 
 ### App Center
 
 #### iOS
 
+* **[Fix]** Remove Keychain permission pop-up on macOS.
 * **[Fix]** Improve encryption security.
 
-### App Center Crashes
+#### Android
+* **[Fix]** Handle incorrect usage of `AppCenter.setLogUrl` API to provide readable error message.
+* **[Fix]** Fix decrypting values that have been stored for more than a year (such as the in-app update token).
+
+### App Center Analytics
 
 #### iOS
+* **[Feature]** Support setting latency of sending events via `[MSAnalytics setTransmissionInterval:]`.
```

Comment:
Please roll back that as we don't support it in Xamarin, same for android

Reply:
Deleted